example/fetch_all_patrons.rb

```diff
@@ -1,4 +1,4 @@
-require_relative 'lib/patreon'
+require_relative '../lib/patreon'
```

Comment:
you can get away from this whole require_relative thing if you `bundle exec` your example

Reply:
I don't know what that means.

Reply:
It seems like we don't even have instructions on how to execute the example, but specifically using `require_relative` here makes for an odd bit of example code. Happy to revisit this in a later pr